deploy/import-upstream.sh

```diff
@@ -1,14 +1,14 @@
 #!/bin/sh
-oc import-image prometheus:v2.11.104 --from=quay.io/prometheus/prometheus --confirm
-oc import-image prometheus-operator:latest --from=quay.io/coreos/prometheus-operator:v0.27.0 --confirm
-oc import-image prometheus-configmap-reloader:latest --from=quay.io/coreos/configmap-reload --confirm
-oc import-image prometheus-config-reloader:latest --from=quay.io/coreos/prometheus-config-reloader:v0.30.1 --confirm
-oc import-image prometheus-alertmanager:v0.15.0 --from=quay.io/prometheus/alertmanager --confirm
+oc import-image prometheus:v2.11 --from=quay.io/openshift/origin-prometheus:v3.11 --confirm
```

Comment:
@csibbitt @pleimer I've migrated to using the OpenShift organization for the container images, and all are now tagged with the `v3.11` image tag.